### DISCUSSION TITLE
New Visualisation for cQube

### DISCUSSION THREAD

**Discussion by abhineetsingh10** [Sunbird-cQube/community#70] [Ideas]
Taking inspiration from the gapminder tool (See the link below)

https://www.gapminder.org/tools/#$chart-type=bubbles&url=v1

I would like to suggest if we can build something similar in cQube. I have written code to recreate the gapminder tool here- https://vizhub.com/abhineetsingh10/081d08de91184a4e9014fced8af570e1?file=main.js

This project uses D3. 

The gapminder tool allows the user to correlate any two available metrics and also see how each entity (State, District etc.) has evolved over the years. Imagine that each bubble represents a state in India and we are able to see how the state is evolving over the years across metrics. Imagine if we do this for UDISE for which we have data over several years. This will help us find states that have made a lot vs no progress over years. In addition, we can bucket states depending on the region in scatter plot and accordingly identify actions that can be taken for list of states in each bucket.

**Comment by surendrasinghs** [comment]
Interesting ideas! I am glad we could discuss a few more of these on the call. Please keep sharing them here.
